docs/apidiffs/current_vs_latest/opentelemetry-instrumentation-api.txt

```diff
@@ -1,2 +1,7 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
++++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) io.opentelemetry.instrumentation.api.instrumenter.PeerServiceResolver  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.instrumentation.api.instrumenter.PeerServiceResolver create(java.util.Map<java.lang.String,java.lang.String>)
+	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isEmpty()
+	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String resolveService(java.lang.String, java.lang.Integer, java.lang.String)
```

Comment:
can `PeerServiceResolver` live in `opentelemetry-api-semconv` artifact? (since it's based on `peer.service` semconv which hasn't been declared stable yet)

Reply:
I don't mind but it is possible to do it and being able to instanciate it once for all at the same time ?
As the peerserviceMapping is provided by the CommonConfig which is not in opentelemetry-api-semconv

Reply:
It is, you'd have to add an implementation dependency on `instrumentation-api-semconv` to the `javaagent-extension-api`. Which seems to be okay to me